Kiosk/Bid Fulfillment/ConfirmYourBidPINViewController.swift

```diff
@@ -65,15 +66,20 @@ class ConfirmYourBidPINViewController: UIViewController {
                         .updateUserCredentials(loggedInProvider)
                         .mapReplace(provider)
                 }
-                .flatMap { provider -> Observable<AuthorizedNetworking> in
+                .flatMap { provider -> Observable<BypassResults> in
```

Comment:
It does feel a bit weird, but nowhere close to being strongly enough that I'm adverse